package.json

```diff
@@ -14,7 +14,7 @@
     "build:types:watch": "ts-node scripts/build-types-watch.ts",
     "build:npm": "lerna run --parallel build:npm",
     "circularDepCheck": "lerna run --parallel circularDepCheck",
-    "clean": "lerna run --parallel clean && lerna clean --yes",
+    "clean": "lerna run --parallel clean && lerna clean --yes && rm -rf eslintcache",
```

Comment:
This will break on windows. Use `rimraf` instead.

Reply:
TIL. Thanks for catching that. Fixed.